config/sys.config

```diff
@@ -90,7 +90,8 @@
            ]},
            {epoch_sync_lager_event, [
              {handlers, [
-               {lager_file_backend, [
+                {lager_console_backend, [{level, info}]},
```

Comment:
Also in epoch_mining_lager_event ?

Reply:
I don't know, we could; this is not related to the changes in that PR though.